Assets/_BuildingBlocks/Credentials/Scripts/ICredentialHandler.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using UnityEngine.Events;
+
+namespace Netherlands3D.Credentials
+{
+    public interface ICredentialHandler
+    {
+        public Uri BaseUri { get; set; } //The base URL to check the credentials for
+        public string UserName { get; set; }
```

Comment:
Waarom gebruik je hier niet een StoredAuthorization instance? Want dat zou het consistent maken in de applicatie; of is er een reden dat dat hier niet gaat werken? (zelfde opmerking is van toepassing op de implementatie van deze interface)

Reply:
Dit is een interface voor de UI, omdat die zowel in de linker flow als de rechter flow kan bestaan

Reply:
Ah, het is meer een ViewModel waaraan je de velden bind, check!